app/processing/siti-agri/filter-payment-requests.js

```diff
@@ -0,0 +1,41 @@
+const buildPaymentRequests = require('./build-payment-requests')
+const { convertToPence, getTotalValueInPence } = require('../../currency-convert')
+
+const paymentRequestSchema = require('./schemas/payment-request')
+const { isInvoiceLineValid } = require('./build-invoice-lines')
+
+const filterPaymentRequest = (paymentRequests, sourceSystem) => {
+  const paymentRequestsCollection = { successfulPaymentRequests: [], unsuccessfulPaymentRequests: [] }
+  buildPaymentRequests(paymentRequests, sourceSystem)
+    .map(x => handlePaymentRequest(x, paymentRequestsCollection))
+  return paymentRequestsCollection
+}
+
+const isPaymentRequestValid = (paymentRequest) => {
+  const validationResult = paymentRequestSchema.validate(paymentRequest, { abortEarly: false })
```

Comment:
A payment request is invalid if there's no invoice lines, however the schema just checks that it's an array.  Can we update to ensure that there is at least one invoice line?